OracleDatabase/SingleInstance/dockerfiles/21.3.0/Dockerfile.xe

```diff
@@ -0,0 +1,87 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2018, 2021 Oracle and/or its affiliates.
+#
+# ORACLE DOCKERFILES PROJECT
+# --------------------------
+# This is the Dockerfile for Oracle Database 21c Express Edition
+# 
+# REQUIRED FILES TO BUILD THIS IMAGE
+# ----------------------------------
+# None
+#
+# HOW TO BUILD THIS IMAGE
+# -----------------------
+# Run: 
+#      $ docker build -t oracle/database:21.4.0-xe -f Dockerfile.xe .
```

Comment:
Oracle XE is 21.3.0 again (18.4.0 was a one-off because it was delayed)

Reply:
Fixed.